composable_kernel/include/tensor_operation/element_wise_operation.hpp

```diff
@@ -12,6 +12,45 @@ struct PassThrough
     __host__ __device__ void operator()(half_t& y, const half_t& x) const { y = x; }
 };
 
+struct Add
+{
+    __host__ __device__ constexpr void
+    operator()(float& dst, const float& src_y, const float& bias) const
```

Comment:
Just use y for output and x for input. Word like "bias" only has meaning in GEMM+Bias fusion, but this "Add" operator will be use in many other places.

operator()(float& y, const float& x0, const float& x1) const

Reply:
https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/78/commits/8a896c4516fe556d043ff39c2ec2e7436d73f3bc

Reply:
Fix in https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/78/commits/8a896c4516fe556d043ff39c2ec2e7436d73f3bc